src/tests/FunctionalTests/Directory.Build.props

```diff
@@ -7,5 +7,44 @@
     <IsFunctionalTest>true</IsFunctionalTest>
   </PropertyGroup>
 
+  <!-- Modeled after Microsoft.Android.Sdk.ILLink.targets -->
```

Comment:
might be useful to link the file. same for the other targets files.

Reply:
Done!